nox/sessions.py

```diff
@@ -544,13 +549,11 @@ def install(self, *args: str, **kwargs: Any) -> None:
                 "A session without a virtualenv can not install dependencies."
             )
         if isinstance(venv, PassthroughEnv):
-            warnings.warn(
-                f"Session {self.name} does not have a virtual environment, "
-                "so use of session.install() is deprecated since it would modify "
-                "the global Python environment. If you're really sure that is "
-                'what you want to do, use session.run("pip", "install", ...) instead.',
-                category=FutureWarning,
-                stacklevel=2,
+            raise ValueError(
```

Comment:
Should I raise something else than `ValueError`?

Reply:
I find it's consistent with other uses of `ValueError` in the Nox codebase, so works for me.